modules/uncertainty/R/ensemble.R

```diff
@@ -393,6 +394,7 @@ write.ensemble.configs <- function(defaults, ensemble.samples, settings, model,
           "site        : ", settings$run$site$name, "\n",
           "site  id    : ", format(settings$run$site$id, scientific = FALSE), "\n",
           "met data    : ", samples$met$samples[[i]], "\n",
+          "soil parameter   : ", samples$soilinitcond$samples[[i]], "\n",
```

Comment:
This line will fail for any model that doesn't have soilinitcond. You should dynamically detect the set of sampled inputs present